nordicNRPlus/parseCbor.ts

```diff
@@ -0,0 +1,27 @@
+import cbor from 'cbor'
+
+export type LocationData = {
+	lat: number
+	lon: number
+	uncertainty: number
+	src: string
+}
+export const parseCbor = (cborData: string): LocationData => {
```

Comment:
```suggestion
export const parseCBORLocationData = (cborData: string): LocationData => {
```

Specific implementation for just one object type.